sources/web/datalab/static/datalab.css

```diff
@@ -115,13 +115,15 @@ body {
   justify-content: space-between;
 }
 #contentArea {
-  height: 100%;
```

Comment:
In my testing I found that the height of the notebook toolbar wasn't correct. The issue was resolved by restoring `height: 100%` under `#contentArea`.

![notebooktoolbarissue](https://cloud.githubusercontent.com/assets/5184014/24332953/8fc8aada-121d-11e7-91de-513962e771b8.png)